### PR TITLE
Marketo Static Lists documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added documentation for the Marketo Static Lists destination
> Sidebar and landing page updates

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/marketo-static-lists.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+- **Munchkin Account ID**: Enter your Munchkin account ID. For more information on obtaining your Munchkin ID, refer to the <Link to="#faq">FAQ</Link> section below.
+- **Client ID**: Enter your Marketo client ID.
+- **Client Secret**: Enter the associated client secret for the above client ID.
+
+<div class="infoBlock">
+For more information on obtaining your Marketo client ID and secret, refer to the <Link to="#faq">FAQ</Link> section below.
+</div>
+
+- **Static List ID**: Enter your Marketo static list ID. RudderStack uses this ID to add or remove leads from the specific list. For more information on obtaining your static list ID, refer to the <Link to="#faq">FAQ</Link> section below.
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/marketo-static-lists.mdx:39]: Can we add one more line here that this Static List Id is used as a fallback if Static List Id is provided in externalId

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/marketo-static-lists.mdx:39]: Added a note addressing this suggestion.

File: docs/destinations/streaming-destinations/marketo-static-lists.mdx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+<div class="infoBlock">
+RudderStack prioritizes the <code class="inline-code">marketoStaticListId</code> provided in the event's <code class="inline-code">externalId</code> over the <strong>Static List ID</strong> dashboard setting. If <code class="inline-code">marketoStaticListId</code> is absent in the payload, RudderStack looks for the static list ID specified in this setting.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/marketo-static-lists.mdx:42]:
```suggestion
RudderStack prioritizes the <code class="inline-code">marketoStaticListId</code> provided in the event's <code class="inline-code">externalId</code> over the <strong>Static List ID</strong> dashboard setting. If <code class="inline-code">marketoStaticListId</code> is absent in <code class="inline-code">externalId</code>, RudderStack looks for the static list ID specified in this setting.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/marketo-static-lists.mdx:42]: @anantjain45823 does this look fine?

Comment (anantjain45823) [reply docs/destinations/streaming-destinations/marketo-static-lists.mdx:42]: Yes

**Review by anantjain45823 — APPROVED** [review]

File: docs/destinations/streaming-destinations/marketo-static-lists.mdx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+- **Client Secret**: Enter the associated client secret for the above client ID.
+
+<div class="infoBlock">
+For more information on obtaining your Marketo client ID and secret, refer to the <Link to="#faq">FAQ</Link> section below.
+</div>
+
+- **Static List ID**: Enter your Marketo static list ID. RudderStack uses this ID to add or remove leads from the specific list. For more information on obtaining your static list ID, refer to the <Link to="#faq">FAQ</Link> section below.
+
+<div class="infoBlock">
+RudderStack prioritizes the <code class="inline-code">marketoStaticListId</code> provided in the event's <code class="inline-code">externalId</code> over the <strong>Static List ID</strong> dashboard setting. If <code class="inline-code">marketoStaticListId</code> is absent in <code class="inline-code">externalId</code>, RudderStack looks for the static list ID specified in this setting.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/marketo-static-lists.mdx:42]:
```suggestion
RudderStack prioritizes the <code class="inline-code">marketoStaticListId</code> provided in the event's <code class="inline-code">externalId</code> over the <strong>Static List ID</strong> dashboard setting. If <code class="inline-code">marketoStaticListId</code> is absent in <code class="inline-code">externalId</code>, RudderStack looks for the static list ID in this setting as it is mandatory to send event data successfully to Marketo.
```

File: docs/destinations/streaming-destinations/marketo-static-lists.mdx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+<img src="../../assets/event-stream-destinations/marketo-services.png" />
+
+4. To create a new service, click **New** > **New Service**.
+5. Enter the **Display Name**. From the **Service** dropdown, select **Custom**.
+6. Under **Settings**, enter the **Description** and select the **API Only User**, as shown. Finally, click **CREATE**.
+
+<img src="../../assets/event-stream-destinations/marketo-services.png" />
+
+<div class="warningBlock">
+Make sure the <strong>API Only User</strong> associated with the API service has the necessary permissions to create or update contacts as wellas the custom activities.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/marketo-static-lists.mdx:136]:
```suggestion
Make sure the <strong>API Only User</strong> associated with the API service has the necessary permissions to create or update contacts as well as the custom activities.
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**